experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -46,27 +46,33 @@ end
 """
    land_albedo(slab_sim::BucketSimulation)
 
-Returns the surface albedo of the earth; 
-a method for the bucket model 
+Returns the surface albedo of the earth;
+a method for the bucket model
 when used as the land model.
 """
 function land_albedo(slab_sim::BucketSimulation)
-    α_land =
-        surface_albedo.(
-            slab_sim.integrator.p.bucket.α_sfc,
-            slab_sim.params.albedo.α_snow,
-            slab_sim.integrator.u.bucket.σS,
-            slab_sim.params.σS_c,
-        )
+    α_land = ClimaLSM.surface_albedo(slab_sim.model, slab_sim.integrator.u, slab_sim.integrator.p)
     return parent(α_land)
 end
 
+"""
+   land_beta(slab_sim::BucketSimulation)
+
+Returns the beta factor of the earth;
+a method for the bucket model
+when used as the land model.
+"""
+function land_beta(slab_sim::BucketSimulation)
```

Comment:
It could be good to clarify that this is referring to the `beta_factor` variable in the [land model](https://github.com/CliMA/ClimaLSM.jl/blob/507ec51e4032593b1cbe3ff10e916d8ee9f06e39/src/Bucket/bucket_parameterizations.jl#L87). Not the `β` variable

Reply:
Added a comment clarifying this!